changelog.md

```diff
@@ -2,3 +2,9 @@
 This file contains all the notable changes done to the Ballerina Email package through the releases.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Changed
+ - [[#1345] Introduce byte stream related functionality to FTP module](https://github.com/ballerina-platform/ballerina-standard-library/issues/1345)
+ - [[#1388] Add verification for tests](https://github.com/ballerina-platform/ballerina-standard-library/issues/1388)
```

Comment:
Is this something we changed or newly added ? If its newly added it should go under Added section

Reply:
This is related to the changed content.